kythe/cxx/indexer/cxx/testdata/proto/testdata.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
```

Comment:
Is this file referenced anywhere?

Reply:
Ach, no, both of those were left over from previous revisions. Also looks like I have to make some fixes for older Bazel.